_button.scss

```diff
@@ -0,0 +1,71 @@
+// button
+// ===
+
+// Configurable Variables
+// ---
+
+//Button Styles
+//---
+
+$button__padding: $vertical-input-padding $horizontal-input-padding !default;
+
+$button__border: if(variable-exists(border), $border, 1px solid $neutral-50) !default;
+$button__border-radius: if(variable-exists(border-radius), $border-radius, 0) !default;
+
+
+$button__font-color: inherit !default;
+$button__font-family: inherit !default;
+$button__font-size: inherit !default;
+
+// Dependencies
+// ---
+
+@import 'stencil-variables/variables';
+@import 'level/level';
+
+
+// Button root
+// ---
+
+.c-button {
+    min-height: $tap-size;
+    display:inline-block;
+    margin: 0;
+    cursor:pointer;
+    background: none;
+
+    text-align:center;
+    border:none;
+    line-height: normal;
+    white-space: normal;
+
+    span {
+        font-size: $button__font-size;
+        font-family: $button__font-family;
+
+        border: $button__border;
+        border-radius: $button__border-radius;
+        padding: $button__padding;
+
+        display:inline-block;
```

Comment:
I’d like to make the visible part of this flexed so that:
- we can set a min-height to the desired visual height
- label and other content (e.g. Icon component) can be centered vertically and horizontally
- label can wrap to two lines if needed